src/app/modules/submission/submission.errors.ts

```diff
@@ -241,3 +241,27 @@ export class ExpectedResponseNotFoundError extends ApplicationError {
     super(message, undefined, ErrorCodes.SUBMISSION_EXPECTED_RESPONSE_NOT_FOUND)
   }
 }
+
+export class MrfReminderInvalidWorkflowStepError extends ApplicationError {
+  constructor(
+    message = 'Unable to send reminder as no valid pending workflow step is found.',
```

Comment:
This message is going to be shown to the user, would prefer that it is less technical

```suggestion
    message = 'Unable to send reminder as workflow step is invalid.',
```